mods/cnc/weapons/missiles.yaml

```diff
@@ -377,16 +377,12 @@ Patriot:
 		RateOfTurn: 20
 		Speed: 341
 		RangeLimit: 30
-		Angle: 88
```

Comment:
The `Missile` projectile doesn't have the `Angle` property. This is a bogus leftover (possibly from when this still was the SSM and the weapon used the Bullet projectile).